pallets/redeem/src/tests.rs

```diff
@@ -1078,3 +1079,431 @@ mod spec_based_tests {
 		})
 	}
 }
+
+#[test]
+fn test_request_redeem_fails_limits() {
+	run_test(|| {
+		crate::Pallet::<Test>::_rate_limit_update(
+			std::option::Option::<u128>::Some(9u128),
+			DEFAULT_COLLATERAL_CURRENCY,
+			7200u64,
+		);
+
+		convert_to.mock_safe(|_, x| MockResult::Return(Ok(x)));
+		<vault_registry::Pallet<Test>>::insert_vault(
+			&VAULT,
+			vault_registry::Vault {
+				id: VAULT,
+				to_be_replaced_tokens: 0,
+				to_be_issued_tokens: 0,
+				issued_tokens: 10,
+				to_be_redeemed_tokens: 0,
+				replace_collateral: 0,
+				active_replace_collateral: 0,
+				banned_until: None,
+				secure_collateral_threshold: None,
+				status: VaultStatus::Active(true),
+				liquidated_collateral: 0,
+			},
+		);
+
+		let redeemer = USER;
+		let amount = 90;
```

Comment:
Can we use `10` as the amount here? I just would prefer to use the value that is as close to the valid threshold as possible. To make it clearer we could also introduce an extra variable at the top like
```
let volume_limit: u128 = 9;
crate::Pallet::<Test>::_rate_limit_update(
			std::option::Option::<u128>::Some(volume_limit),
			DEFAULT_COLLATERAL_CURRENCY,
			7200u64,
		);
...
...
let amount =  volume_limit + 1;
```

This way it is very clear why we choose which amount.

Reply:
done